test/trigger_test.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+
+# NOTE: this script will execute from the repository root when called on jenkins
+
+VAULT_TOKEN=$1
+INPUTS_JSON=$2
+WDL_FILE=$3
+DEPENDENCIES_JSON=$4
+WD=$(pwd)
+echo $(ls)
+echo ${WD}
+
+echo "Rendering templates"
+bash ${WD}/test/config/render-ctmpls.sh "dev" "${VAULT_TOKEN}"
+
+echo "Setting Cromwell environmental variables"
+CROMWELL_SECRETS="${WD}/test/cromwell-secrets.json"
+CROMWELL_USER=$(cat "${CROMWELL_SECRETS}" | jq -r .cromwell_user)
+CROMWELL_PASSWORD=$(cat "${CROMWELL_SECRETS}" | jq -r .cromwell_password)
+
+OPTIONS_FILE="https://raw.githubusercontent.com/HumanCellAtlas/skylab/master/test/options.json"
```

Comment:
would you want to be grabbing the options file from github master branch or the branch being tested?  (it'll likely not change much at all either way)

Reply:
I think master; it just specifies no call caching.